services/node/node-base.js

```diff
@@ -5,13 +5,9 @@ const NPMBase = require('../npm/npm-base')
 const keywords = ['npm']
 
 module.exports = class NodeVersionBase extends NPMBase {
-  static get category() {
-    return 'platform-support'
-  }
+  static category = 'platform-support'
 
-  static get route() {
-    return this.buildRoute(`node/${this.path}`, { withTag: true })
-  }
+  static route = this.buildRoute(`node/v-lts`, { withTag: true })
```

Comment:
the LTS and Current services have different routes, so they need to define their respective routes. Can't hardcode this in the parent class here

Reply:
@calebcartwright  yes trying that.its in review phase

Reply:
Note that you can mark a PR as a draft if you aren't ready for a review, but otherwise as maintainers we'll review and provide feedback on open PRs especially updates to existing ones.

If you're just trying to get the tests to run, also note that you can run them locally `npm run test:services -- --only=node,nexus` in this case

Reply:
@calebcartwright  wohooo it worked. Don't merge it pushing other modules